Pollination-apps/chart.md

```diff
@@ -0,0 +1,40 @@
+# Add a chart to your Pollination app
+
```

Comment:
We can have a short introduction:

In this tutorial, you will learn how to add a Chart to your app by using the `ladybug-charts` library. We will use an EPW weather file for this example and visualize the dry bulb temperature as a heatmap. Ladybug-charts provides several other graphs such as X, Y, Z!